### ISSUE TITLE
 Issue with Setting Leader Key and Ex Command in Vim on MacBook Air M1 E480: No match: g E480: No match: keymap

### ISSUE THREAD

**Issue by Daviddlh1** [ThePrimeagen/init.lua#117]
Hi,

I'm trying to follow a video tutorial, but I encounter an error when I attempt to set a new leader key and a new Ex command:

<img width="546" alt="Screenshot 2024-05-23 at 8 01 07 AM" src="https://github.com/ThePrimeagen/init.lua/assets/84351400/e4241f2a-c414-4aec-872e-7f1c7d829e30">

The error message I receive is:

<img width="561" alt="Screenshot 2024-05-23 at 8 00 54 AM" src="https://github.com/ThePrimeagen/init.lua/assets/84351400/f45e4375-510f-4f77-aee2-db3e469b1d8f">

According to some Reddit posts, this issue occurs when using a version prior to 0.7. However, when I run :version in Vim, it indicates that I'm using version 0.11, so I believe I shouldn't be experiencing this issue:

<img width="646" alt="Screenshot 2024-05-23 at 8 02 07 AM" src="https://github.com/ThePrimeagen/init.lua/assets/84351400/5ca6e2af-7a0c-4731-b9e2-eefeb0265619">

Does anyone know what could be causing this issue? 
I'm using a MacBook Air M1 so I'm not sure if the Apple silicons have this error for some reason (which I strongly doubt but who knows)

Does anyone know what might be causing this issue? For reference, I'm using a MacBook Air M1. I doubt it's related to the Apple silicon, but I'm open to any suggestions.

Thank you!

**Comment by ewaddd** [comment]
You must pass a reference to the function `vim.cmd.Ex` instead of call it `vim.cmd.Ex`

**Comment by Daviddlh1** [comment]
Hi @ewaddd,

Thanks for your answer. I tried your solution, but the error is still the same.

<img width="624" alt="Screenshot 2024-05-23 at 1 15 27 PM" src="https://github.com/ThePrimeagen/init.lua/assets/84351400/52e7211c-a359-40da-80ed-310f87353a3f">

I have never used Lua before, but the error message makes me think that the Vim object itself might not be there. It says there is no match for "g," which I assume refers to "vim.g." It's like I'm trying to access a property of a null object. However, I'm not really sure about that.

**Comment by Daviddlh1** [comment]
Hi everyone,

I finally solved the issue. It turns out that I was trying to configure Vim with Lua instead of Neovim. In the tutorial video, Prime constantly using the command vim rather than nvim (jajajaj maybe he has an alias or my dyslexia confused me), so I was trying to configure it in vim. However, once I started using the :so command in nvim, all the configurations began to work properly.

Thank you for your suggestions and help!
